core/src/main/java/com/novoda/noplayer/exoplayer/mediasource/ExoPlayerAudioTrackSelector.java

```diff
@@ -0,0 +1,62 @@
+package com.novoda.noplayer.exoplayer.mediasource;
+
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.source.TrackGroup;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.MappingTrackSelector;
+import com.google.android.exoplayer2.trackselection.TrackSelection;
+import com.novoda.noplayer.PlayerAudioTrack;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExoPlayerAudioTrackSelector {
```

Comment:
Previously named `ExoPlayerTrackSelector` but needed to introduce another layer for methods that could not be mocked or extended 😢  `ExoPlayerTrackSelector` is now the wrapper around non-mockable methods and this `ExoPlayerAudioTrackSelector` handles all of the audio track selection (useful naming 👅 )